db/migrate/20170509235403_add_relationships.rb

```diff
@@ -0,0 +1,4 @@
+class AddRelationships < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
What's going on with this empty change method in this migration? Seems like you could delete this migration if it's not doing anything...